src/Woman.java

```diff
@@ -0,0 +1,36 @@
+public class Woman extends Person{
+    private String maidenlastName;
+
+    public Woman(String firstName, String lastName, int age) {
+        super(firstName, lastName, age);
+        this.maidenlastName = lastName;
+    }
+
+    @Override
+    public boolean isRetired() {
+        return (this.getAge()>=60);
+    }
+
+    public void registerPartnership(Person partner){
```

Comment:
This method must be defined for both Woman and Man

Reply:
Ok